tutorials/cp-starter-integration-cpi-onboard-subscribe/cp-starter-integration-cpi-onboard-subscribe.md

```diff
@@ -96,40 +115,52 @@ To provision your SAP Cloud Platform Integration tenant, you must first subscrib
 [ACCORDION-END]
 
 [ACCORDION-BEGIN [Step 4: ](Provision SAP Cloud Platform Integration tenant)]
-1. Launch SAP Cloud Platform Integration provisioning application by choosing **`<subaccount-name>`** > **Subscriptions** > **Go to Application** in the **Process Integration** tile.
+1. Launch SAP Cloud Platform Integration provisioning application by choosing **`<subaccount-name>`** > **Subscriptions**.
+
+2.  Click **Go to Application** in the **Process Integration** tile.
 
     ![Access cloud integration provisioning application](5.1.access-prov-app.png)
 
-2. Login to the provisioning application by providing the same email ID and password that you used for creating the trial account. Choose **Log On**.
+3. Login to the provisioning application by providing the same email ID and password that you used for creating the trial account. Choose **Log On**.
 
     ![Login to provisioning application](5.2.login-prov-app.png)
 
-3. Choose **Provision** to provision a SAP Cloud Platform Integration tenant.
+4. Choose **Provision** to provision a SAP Cloud Platform Integration tenant.
 
     ![Provision SAP Cloud Platform Integration tenant](5.3.provision-cpi.png)
 
-4. You see a tenant URL after the provisioning is complete. This is the URL for your SAP Cloud Platform Integration tenant.
+5. You see a tenant URL after the provisioning is complete. This is the URL for your SAP Cloud Platform Integration tenant.
 
     ![Access tenant URL](5.4.access-tenant-url.png)
 
-    >**IMPORTANT:** Please wait for the provisioning process to complete before you proceed to the next step. The service instance and key creation will only work after your tenant has been provisioned.
+    >**IMPORTANT:** Please wait for the provisioning process to complete before you proceed to the next step. This can take several minutes.
+    The service instance and key creation will only work after your tenant has been provisioned.
 
     Click on the URL to launch the SAP Cloud Platform Integration application. You can now create integration packages and integration flows.
 
-    Please bookmark or save this URL. This will be the URL for your SAP Cloud Platform Integration application.
+    Bookmark or save this URL. This will be the URL for your SAP Cloud Platform Integration application.
 
 
 [DONE]
 [ACCORDION-END]
 
 [ACCORDION-BEGIN [Step 5: ](Create service instance and key)]
+## Details:
```

Comment:
Why is there a header here? Please remove.

Also, that's quite a lot of introductory text. I think you can cut it down by half. Also, "In the previous step, you have configured" --> use the past simple tense -- "you configured".

Also, "in this step, you do enable" --> delete the word "do" here.

Reply:
Hi @thecodester, 

I have corrected the suggestion but the long introductory text required here. Since our idea is to elaborate on the concepts and make end-user to understand more. These are the few changes we have enhanced in this tutorial.